tests/Temporalio.Tests/WorkerAssertionExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Temporalio.Api.History.V1;
+using Temporalio.Client;
+using Xunit;
+
+namespace Temporalio.Tests
+{
+    public static class WorkerAssertionExtensions

```

Comment:
I wanted to re-use some of these methods that were originally in `WorkflowWorkerTests.cs` in `TemporalWorkerServiceTests.cs`, so I pulled them out into extension methods here.

Reply:
Done